prime-angular-frontend/src/app/modules/paper-enrolment/pages/demographic-page/demographic-page.component.ts

```diff
@@ -132,8 +133,4 @@ export class DemographicPageComponent extends AbstractEnrolmentPage implements O
 
     this.routeUtils.routeRelativeTo(nextRoutePath);
   }
-
-  protected onSubmitFormIsInvalid(): void {
-    this.showAddressFields = true;
-  }
```

Comment:
previously this is used to show validations when address is empty. it seems that now we lost the ability to show the validation errors if leaving the address field empty after clicking `save and continue` button.

Reply:
So I added a new boolean to force show when the form is invalid. I had to change the previous boolean to a getter since we now have more than one address and we  want to check each one for whether or not we should display the form fields. 

I don't think it's possible to have an additional address field be empty and its fields hidden, so in the method `onSubmitFormIsInvalid` I am only forcing the main address fields to be shown, instead of the main one and all additional addresses

Reply:
somehow after the fix it's not saving the additional address, only first address is saved.

Reply:
Fixed.